test/bats/tests/constraints/all_ns_must_have_gatekeeper.yaml

```diff
@@ -3,6 +3,7 @@ kind: K8sRequiredLabels
 metadata:
   name: ns-must-have-gk
 spec:
+  enforcementAction: dryrun
```

Comment:
can we keep this as and create a separate constraint with dryrun? then in the e2e, run the dryrun one first, expecting success, then deploy the deny one, expecting failure.

Reply:
+1

Reply:
updated